selfdrive/car/honda/carstate.py

```diff
@@ -322,6 +322,10 @@ def update(self, cp, cp_cam):
       self.stock_hud = cp_cam.vl["ACC_HUD"]
       self.stock_brake = cp_cam.vl["BRAKE_COMMAND"]
 
+    if self.CP.carFingerprint in (CAR.CRV_5G):
```

Comment:
Somehow this works, but you should probably add a comma.

```suggestion
    if self.CP.carFingerprint in (CAR.CRV_5G, ):
```